terraform/examples/module_enabled/README.md

```diff
@@ -0,0 +1,74 @@
+
+# Example for the `module_enabled` feature
```

Comment:
Let's use something more expressive here, e.g. `How to dynamically enable modules using the module_enabled toggle.`

I also believe that we can shorten syntax, e.G. `enable = true`

Reply:
shorter syntax might conflict with resource `enabled` fields... since this is actually just a workaround for missing terraform features i would avoid any potential clash in naming by prefixing..

Reply:
fixed heading